Common/Tests/MockVsTests/MockVsRunningDocumentTable.cs

```diff
@@ -19,11 +19,7 @@
 using Microsoft.VisualStudio.Shell.Interop;
```

Comment:
Whoops, guess I did touch this one file in common too. I'll revert it if needed.